requirements.txt

```diff
@@ -1,4 +1,6 @@
-# This can not be installed from pip, on Fedora you need to install rpm-python{,3}
+# This can not be installed from pip, on Fedora you need to install
+# python{2,3}-rpm RPM including rpm-python pip package.
```

Comment:
Thanks for the checking again. I fixed the commit message. And rebased by myself now.